### ISSUE TITLE
Wheel support

### ISSUE THREAD

**Issue by kevin-brown** [JoshData/python-email-validator#15]
http://pythonwheels.com/

Right now, only the `tar.gz` file is being distributed on PyPI for [email_validator](https://pypi.python.org/pypi/email_validator), which isn't _that much_ of an issue, but it does add some time to installing the package.

This package should be compatible with the Wheel format, considering it doesn't appear to have any C dependencies and it is compatible with both Python 2 and 3. As a result, you should only need to generate a universal wheel and then everyone (on all systems) will get the ability to install email_validator with just the wheel, without having to do any extra work.

**Comment by JoshData** [comment]
Great idea. Any chance you could submit a pull request to do it?

On August 31, 2017 11:29:36 AM EDT, Kevin Brown <notifications@github.com> wrote:
>http://pythonwheels.com/
>
>Right now, only the `tar.gz` file is being distributed on PyPI for
>[email_validator](https://pypi.python.org/pypi/email_validator), which
>isn't _that much_ of an issue, but it does add some time to installing
>the package.
>
>This package should be compatible with the Wheel format, considering it
>doesn't appear to have any C dependencies and it is compatible with
>both Python 2 and 3. As a result, you should only need to generate a
>universal wheel and then everyone (on all systems) will get the ability
>to install email_validator with just the wheel, without having to do
>any extra work.
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/JoshData/python-email-validator/issues/15

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by svisser** [comment]
It should be possible by running:

```bash
pip3 install wheel
python3 setup.py bdist_wheel --universal
```

**Comment by JoshData** [comment]
Awesome, I just published 1.0.3 as a wheel (no other changes from 1.0.2)!
